pom.xml

```diff
@@ -577,7 +577,7 @@
       <dependency>
         <groupId>${project.groupId}</groupId>
         <artifactId>netty-tcnative</artifactId>
-        <version>1.1.32.Fork1</version>
+        <version>1.1.32.Fork2-SNAPSHOT</version>
```

Comment:
I also would prefer depending on a release version.  Releasing `netty-tcnative` is relatively easier than releasing Netty (no release notes, etc), so how about cutting a release first, @normanmaurer ? :-)